jobs/aws/eks-anywhere-build-tooling/harbor-presubmits.yaml

```diff
@@ -56,9 +56,11 @@ presubmits:
           requests:
             memory: "16Gi"
             cpu: "4"
+            ephemeral-storage: "8Gi"
           limits:
             memory: "16Gi"
             cpu: "4"
+            ephemeral-storage: "8Gi"
```

Comment:
I think you should bump this down to `4Gi` I think that's the limit per container.